grpc/creds/creds.go

```diff
@@ -154,33 +160,34 @@ func (tc *serverTransportCredentials) validateClient(peerState tls.ConnectionSta
 
 	// Since we call `conn.Handshake()` before `validateClient` and ensure
 	// a non-error response we don't need to validate anything except the presence
-	// of an accepted SAN in the leaf entry of `PeerCertificates`. The tls
+	// of an acceptable SAN in the leaf entry of `PeerCertificates`. The tls
```

Comment:
I can't add a comment on LIne 137, but could you do me the favour of also fixing the typo in the `validateClient` comment:
>  // have provided the `tls.ConnectionState` from returned from a validate (e.g.

Thanks!